Client.xcodeproj/project.pbxproj

```diff
@@ -9610,7 +9598,6 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				LIBRARY_SEARCH_PATHS = "$(inherited)";
```

Comment:
here we previously didn't add sqlcipher and this scheme did not compile, now all seaarch paths are at top level, which makes it easier to look for such cases